base/setup/usetup/lang/es-ES.h

```diff
@@ -465,35 +465,35 @@ static MUI_ENTRY esESDevicePageEntries[] =
     {
         6,
         19,
-        "Puede modificar la configuraci\242n con las teclas ARRIBA y ABAJO",
+        "Puede escoger la configuraci\242n de harware con las teclas",
```

Comment:
"hardware"?

Reply:
Yes, it should be "hardware".

Reply:
I have to fix two more lines. The jumping of one line. I will update it as soon as possible.